src/components/Input.css

```diff
@@ -0,0 +1,52 @@
+.input-box {
+  margin: 20px 0;
+  padding: 20px;
+  width: 250px;
+  background-color: #f2f0f0;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  border: solid black 1px;
+  box-shadow: 10px 10px 2px 3px black;
+}
```

Comment:
Since you have a textarea in your form and it has a  default settung that enables the user to change the size of the box, you might want to take this away. You could use "resize: none;"